monitoring/bentoml-plugins-arize/pyproject.toml

```diff
@@ -48,3 +48,6 @@ include = ["/src", "/tests"]
 [tool.hatch.build.targets.wheel]
 packages = ["src/bentoml_plugins"]
 exclude = ["src/bentoml_plugins/arize/BUILD"]
+
+[tool.pyright]
+pythonVersion = "3.11"
```

Comment:
we already set in this in the pyproject.toml in the root dir.

Reply:
I see. But it seems not bad to have it here two